src/web/components/Dateline.tsx

```diff
@@ -17,18 +17,89 @@ const dateline = css`
     margin-bottom: 6px;
 `;
 
-const description = css`
-    ${visuallyHidden}
+const secondaryDateWidth = css`
+    ${from.leftCol} {
+        width: 90%;
+    }
 `;
 
+// Checkbox functionality
+const toggleClass = css`
+    input[type='checkbox'] {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        opacity: 0;
+        display: none;
+    }
+
+    input[type='checkbox']:checked ~ span ~ p {
+        max-height: 80px;
+    }
+`;
+
+const pCheckboxStyle = css`
+    p {
+        max-height: 0;
+        overflow: hidden;
+        transition: max-height 0.4s ease;
+    }
+`;
+
+const labelStyles = css`
+    label {
+        cursor: pointer;
+    }
+    label {
+        position: relative;
+        display: block;
+    }
+    label:hover {
+        text-decoration: underline;
+    }
+    label::before {
+        content: '';
+        position: absolute;
+        width: 0;
+        height: 0;
+        top: 50%;
+        left: 10px;
+    }
+`;
+
+// At the moment there is no way to tell from primaryDateline and secondaryDatline
+// what the prepend should be (either 'Last modified on' or 'First published on ')
 export const Dateline: React.FC<{
-    dateDisplay: string;
+    primaryDateline: string;
     descriptionText: string;
-}> = ({ dateDisplay, descriptionText }) => (
-    <>
-        <div className={dateline} role="textbox">
-            <span className={description}>{descriptionText} </span>
```

Comment:
What happened to the descriptionText here?

Reply:
I think the idea for descriptionText was to use it to show "Last modified on" and "First published on", but with the data coming through as strings there's no simple way of determining which to use. I'll remove descriptionText for now.

Reply:
Right, I see what you mean, so I guess we need to know that from the data model..?

Reply:
On some articles the publication date is back-dated so the primary date is actually after the secondary date (https://www.theguardian.com/uk-news/2020/aug/12/reports-of-serious-injuries-after-train-derails-scotland-stonehaven). It may be easier to pass these dates as timestamps as they can be checked against each other mathematically to determine if the date is before or after.

Reply:
I think this is why we originally wanted to pass the full dateline with the 'last modified' 'first published' etc text intact 🤦

Reply:
Yeah, I had a play with frontend yesterday and it's pretty straightforward to get the full string in secondaryDateLine if needed.